app/scoring/page.tsx

```diff
@@ -1,9 +1,59 @@
-export default function Scoring() {
-    var currentSelectedContestant = "Jacob"
+import { getData, wikiUrl } from "../utils/wikiScraping"
 
+export default async function Scoring() {
+
+    const wikiData = await getData()
```

Comment:
I believe this conflicts with the API rework changes that have been pulled.

Reply:
LOL, it absolutely does, plus at this point there is some unit testing stuff I would like to pull in, in addition to a slight rework of the aforementioned API rework 😓

Reply:
Alright, this should be addressed now in #21  and the deal before it (rework of the aforementioned Api rework), #20 and so this code is no longer up to date and has been replaced by code that works.